src/table/resizer/index.tsx

```diff
@@ -24,6 +24,10 @@ const AUTO_GROW_START_TIME = 10;
 const AUTO_GROW_INTERVAL = 10;
 const AUTO_GROW_INCREMENT = 5;
 
+export function Divider({ className }: { className?: string }) {
+  return <span className={clsx(styles.divider, styles['divider-disabled'], className)} />;
```

Comment:
Looks like the `divider-disabled` class is applied unconditionally, is this intended?

Reply:
Yes, the table header renders either Resizer or Divider (when resizableColumns=false).

Reply:
But what is the point of the `divider-disabled` class if it is going to be always present?

So this selector
```
.divider.divider-disabled
```
is equivalent to
```
.divider
```

Reply:
The .divider class is also assigned for the separator element of the Resizer component.